### DISCUSSION TITLE
94925

### DISCUSSION THREAD

**Discussion by RickyITA** [sebr/bhyve-home-assistant#112] [Q&A]
Hi, I hope the integration of the 94925 model will arrive soon as well!

**Comment by sebr** [comment]
Have you tried it yet? It probably does work :)

**Reply by RickyITA** [reply]
Yes I tried but it doesn't work. i downloaded from HACS, installed, but then does not find the integration in the list: devices and services, add integration but it is not there!

**Reply by sebr** [reply]
Okay, can you please [raise an issue](https://github.com/sebr/bhyve-home-assistant/issues/new?assignees=sebr&labels=&template=new_device.md&title=) with the debug logs so I can take a look?

Thanks

**Reply by RickyITA** [reply]
I have no log because I can't even find the integration. I am attaching the screenshots
In the first photo you see that the integration is correctly downloaded from HACS,but in the second you see that it is not in the list
![1](https://user-images.githubusercontent.com/2
![2](https://user-images.githubusercontent.com/25688939/155672387-e9443a62-c8a2-452b-b00a-dd37f8668903.png)
5688939/155672378-e31897db-7546-4f84-bd0c-76707c503822.png)
 of those available

**Reply by sebr** [reply]
Right... there is no support for config via the UI. You need to configure with `configuration.yaml`

Please refer to the readme

**Reply by RickyITA** [reply]
Ah ok sorry! So then I find the entity to configure?

**Comment by RickyITA** [comment]
![1](https://user-images.githubusercontent.com/25688939/160548196-dd74490f-0584-4fc3-8e4c-c0e7b895d4dc.jpg)

**Reply by RickyITA** [reply]
I have inserted the switches but they do not work, even if active the irrigation does not actually start
